test/prog/dftb+/tests

```diff
@@ -90,6 +90,13 @@ scc/C60                                #? MPI_PROCS <= 4
 analysis/C60_ESP                       #? MPI_PROCS <= 4
 spin/H2O                               #? MPI_PROCS <= 1
 
+#! This block of tests are for code features capable of producing
+#! unphysical results
+dangerous/C_HardN_defaultSKinterp      #? MPI_PROCS <= 4
+dangerous/C_HardY_defaultSKinterp      #? MPI_PROCS <= 4
+dangerous/C_HardN_oldSKinterp          #? MPI_PROCS <= 4
+dangerous/C_HardY_oldSKinterp          #? MPI_PROCS <= 4
+
```

Comment:
Maybe,  `C_SkCutoff_hard_default`, `C_SkCutoff_soft_default` would be more descriptive (the main feature of the tests is the sk-cutoff).

Reply:
Renamed